blazor/pager/getting-started.md

```diff
@@ -260,62 +318,108 @@ In the following sample, Pager component is integrated with ListView component.
-        </SfPager>
-    </div>
+    <SfPager PageSize=5 NumericItemsCount=4 TotalItemsCount=25>
+    </SfPager>
+
+</div>
+@code {
+    public int SkipValue;
+    public int TakeValue = 5;
+    List<DataModel> ListData = new List<DataModel>();
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        ListData.Add(new DataModel { Name = "Nancy, Berlin, France" });
+        ListData.Add(new DataModel { Name = "Andrew, Madrid, Germany" });
+        ListData.Add(new DataModel { Name = "Janet, London, Brazil" });
+        ListData.Add(new DataModel { Name = "Margaret, Marseille, Belgium" });
+        ListData.Add(new DataModel { Name = "Steven, Cholchester, Switzerland" });
+        ListData.Add(new DataModel { Name = "Laura , Tsawassen, Venezuela" });
+        ListData.Add(new DataModel { Name = "Robert, Tacoma, Austria" });
+        ListData.Add(new DataModel { Name = "Michael, Redmond, Mexico" });
+        ListData.Add(new DataModel { Name = "Albert, Kirkland, USA" });
+        ListData.Add(new DataModel { Name = "Nolan, London, Sweden" });
+        ListData.Add(new DataModel { Name = "Jennifer, Berlin, Finland" });
+        ListData.Add(new DataModel { Name = "Carter, Madrid, Italy" });
+        ListData.Add(new DataModel { Name = "Allison, Marseille, Spain" });
+        ListData.Add(new DataModel { Name = "John, Tsawassen, UK" });
+        ListData.Add(new DataModel { Name = "Susan, Redmond, Ireland" });
+        ListData.Add(new DataModel { Name = "Lydia, Cholchester, Portugal" });
+        ListData.Add(new DataModel { Name = "Kelsey, London, Canada" });
+        ListData.Add(new DataModel { Name = "Jessica, Kirkland, Denmark" });
+        ListData.Add(new DataModel { Name = "Robert, Berlin, Austria" });
+        ListData.Add(new DataModel { Name = "Shelley, Tacoma, Poland" });
+        ListData.Add(new DataModel { Name = "Vanjack, Tsawassen, Norway" });
+        ListData.Add(new DataModel { Name = "shelley, Cholchester, Argentina" });
+        ListData.Add(new DataModel { Name = "Lydia, Kirkland, Finland" });
+        ListData.Add(new DataModel { Name = "Jessica, Madrid, Sweden" });
+        ListData.Add(new DataModel { Name = "Nolan, London, UK" });
+        ListData.Add(new DataModel { Name = "Jennifer, Redmond, Italy" });
+    }
+    public class DataModel
+    {
+        public string Name { get; set; }
+        public string Id { get; set; }
+    }
+}  
+```
+
+Through the navigation of pager items, you can view the items in the list view page by page. This can be achieved by using the [Click](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfPager.html#Syncfusion_Blazor_Navigations_SfPager_Click) event of the Pager. In the `Click` event of the Pager, SkipValue and TakeValue are calculated using the `PageSize` property and arguments of the `Click` event(CurrentPage, PreviousPage). Based on these details, you can view the items in the list view page by page.
```

Comment:
Through the navigation of the pager items, view the items in the list view page by page. This can be achieved by using the [Click](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfPager.html#Syncfusion_Blazor_Navigations_SfPager_Click) event of the Pager. In the `Click` event of the Pager, the SkipValue and TakeValue are calculated using the `PageSize` property and arguments of the `Click` event (CurrentPage, PreviousPage). Based on these details, view the items in the list view page by page.